doc/tutorials/faq.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Frequently Asked Questions (FAQ)
+layout: doc.hbs
+---
+
+# Frequently Asked Questions
+
+Certain questions arise more often than others at the various places users can
+ask for help. This document tries to list some of the common questions from
+the [mailing list](https://groups.google.com/forum/#!forum/ol3-dev), the 
+[github](https://github.com/openlayers/ol3/issues) issues and [Stack 
+Overflow](http://stackoverflow.com/questions/tagged/openlayers-3).
+
```

Comment:
Again, I wouldn't list these at all for this doc. I know these are used as sources of information, but this is kind of historical.